docs/config/README.md

```diff
@@ -698,6 +699,76 @@ truncation_length = 8
 truncation_symbol = "…/"
 ```
 
+## Disk Used
+
+The `disk_used` module shows disk used in current directory or any disk specified.
```

Comment:
Please rename the module to `disk_usage` to align with the `memory_usage` module

Reply:
👍